### DISCUSSION TITLE
Can't uninstall oh-my-posh installed from mstore

### DISCUSSION THREAD

**Discussion by llricci** [JanDeDobbeleer/oh-my-posh#3719] [Troubleshoot] [answered]
I was trying to uninstall `oh-my-posh` for a reinstall, I deleted all the folders which are used by the program and the `$PROFILE` file. However its still appeares as installed in `winget list`. As show below.

```
PS C:\Users\REDECTED>  winget list powershell
Nome                       ID                        Versão       Disponível Origem
------------------------------------------------------------------------------------
PowerShell                 Microsoft.PowerShell      7.3.3.0                 winget
Oh My Posh version 14.25.1 XP8K0HKJFRXGCK            14.25.1      14.27.0    msstore
Terminal do Windows        Microsoft.WindowsTerminal 1.16.10262.0            winget
```
And if I try to use `winget remove --id XP8K0HKJFRXGCK`, it says the app was not found, meaning they should be uninstalled. How to remove it? I downloaded by the `mstore`.

**Comment by JanDeDobbeleer** [comment]
@lucasricci installations on Windows using either Store or winget need to be uninstalled either via the winget CLI or the Add/Remove Program functionality. Removing folders manually will indeed result in tools being confused.

**Reply by llricci** [reply]
What should I do then?

**Reply by llricci** [reply, marked as answer]
nv, already fix it by running the command to install `oh-my-posh` via `winget` and uninstalling it with `winget remove --id XP8K0HKJFRXGCK`.
